build/Build.Pack.cs

```diff
@@ -520,7 +521,16 @@ string ConstructRedHatPackageFilename(string packageName, string architecture)
             const string description = "The deployment agent that is installed on each machine you plan to deploy to using Octopus.";
             const string author = "Octopus Deploy";
             const string title = "Octopus Tentacle cross platform bundle";
-            OctoCliTool($@"pack --id=Octopus.Tentacle.CrossPlatformBundle --version={FullSemVer} --basePath={workingDirectory} --outFolder={ArtifactsDirectory / "nuget"} --author=""{author}"" --title=""{title}"" --description=""{description}""");
+
+            OctopusTasks.OctopusPack(o => o
+                .SetId("Octopus.Tentacle.CrossPlatformBundle")
+                .SetVersion(FullSemVer)
+                .SetBasePath(workingDirectory)
+                .SetOutputFolder(ArtifactsDirectory / "nuget")
+                .SetAuthors(author)
+                .SetTitle(title)
+                .SetDescription(description)
+            );
```

Comment:
The string interpolation led to issues with extraneous quotes. Using the built-in Nuke build support is more reliable